XenAdmin/Alerts/Types/ClientUpdateAlert.cs

```diff
@@ -133,28 +133,14 @@ public static void DownloadAndInstallNewClient(ClientUpdateAlert updateAlert, IW
                 break;
             }
 
-            if (currentTasks)
+            if (!currentTasks)

```

Comment:
I think this one is supposed to be `if(currentTasks)`

This way we're prompted even when there are no actions:
![image](https://user-images.githubusercontent.com/32554698/161002069-7715c820-0ce7-464d-85fd-6e5b7e5102a4.png)

Reply:
woops that was a bit of test  cdoe thanks for the spot